tests/codegen/call_kstack.cpp

```diff
@@ -14,7 +14,9 @@ TEST(codegen, call_kstack)
   // generate the same IR
   test("kprobe:f { @x = kstack(); @y = kstack(6) }", result);
   test("kprobe:f { @x = kstack(perf); @y = kstack(perf, 6) }", result);
-  test("kprobe:f { @x = kstack(perf); @y = kstack(bpftrace) }", result);
+  test(
+      "kprobe:f { @x = kstack(perf); @y = kstack(bpftrace); @z = kstack(raw) }",
+      result);
```

Comment:
I don't think that this is going to generate the same codegen as the previous two (three `kstack` calls vs two). Maybe rather add:
```cpp
test("kprobe:f { @x = kstack(raw); @y = kstack(raw, 6) }", result);
```